cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagement.scala

```diff
@@ -61,8 +62,9 @@ object ClusterHttpManagementRoutes extends ClusterHttpManagementHelper {
         }
 
         val leader = cluster.readView.leader.map(_.toString)
+        val oldest = cluster.state.members.toSeq.sorted(Member.ageOrdering).headOption.map(_.address.toString)
```

Comment:
I think this should also filter to only include members with status `Up`
```
.filter(_..status == MemberStatus.Up)
```

That is what the `ClusterSingletonProxy` is doing.